2023-06-26 13:26:16,209 INFO    MainThread:45508 [wandb_setup.py:_flush():76] Current SDK version is 0.15.4
2023-06-26 13:26:16,209 INFO    MainThread:45508 [wandb_setup.py:_flush():76] Configure stats pid to 45508
2023-06-26 13:26:16,209 INFO    MainThread:45508 [wandb_setup.py:_flush():76] Loading settings from /home/fangkangmi/.config/wandb/settings
2023-06-26 13:26:16,210 INFO    MainThread:45508 [wandb_setup.py:_flush():76] Loading settings from /home/fangkangmi/Documents/workspace/sentiment-analysis/wandb/settings
2023-06-26 13:26:16,210 INFO    MainThread:45508 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-06-26 13:26:16,210 INFO    MainThread:45508 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-06-26 13:26:16,210 INFO    MainThread:45508 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-06-26 13:26:16,210 INFO    MainThread:45508 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-06-26 13:26:16,210 INFO    MainThread:45508 [wandb_init.py:_log_setup():507] Logging user logs to /home/fangkangmi/Documents/workspace/sentiment-analysis/wandb/run-20230626_132616-xj0nyc6c/logs/debug.log
2023-06-26 13:26:16,210 INFO    MainThread:45508 [wandb_init.py:_log_setup():508] Logging internal logs to /home/fangkangmi/Documents/workspace/sentiment-analysis/wandb/run-20230626_132616-xj0nyc6c/logs/debug-internal.log
2023-06-26 13:26:16,210 INFO    MainThread:45508 [wandb_init.py:init():547] calling init triggers
2023-06-26 13:26:16,211 INFO    MainThread:45508 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-06-26 13:26:16,211 INFO    MainThread:45508 [wandb_init.py:init():571] re-initializing run, found existing run on stack: uu5jztpn
2023-06-26 13:26:16,212 INFO    MainThread:45508 [wandb_run.py:_finish():1890] finishing run fangkangmi/sentiment-analysis/uu5jztpn
2023-06-26 13:26:16,213 INFO    MainThread:45508 [jupyter.py:save_history():445] not saving jupyter history
2023-06-26 13:26:16,213 INFO    MainThread:45508 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-06-26 13:26:16,213 INFO    MainThread:45508 [wandb_init.py:_jupyter_teardown():435] cleaning up jupyter logic
2023-06-26 13:26:16,213 INFO    MainThread:45508 [wandb_run.py:_atexit_cleanup():2124] got exitcode: 0
2023-06-26 13:26:16,213 INFO    MainThread:45508 [wandb_run.py:_restore():2107] restore
2023-06-26 13:26:16,213 INFO    MainThread:45508 [wandb_run.py:_restore():2113] restore done
2023-06-26 13:28:37,929 INFO    MainThread:45508 [wandb_init.py:init():596] starting backend
2023-06-26 13:28:37,929 INFO    MainThread:45508 [wandb_init.py:init():600] setting up manager
2023-06-26 13:28:37,931 INFO    MainThread:45508 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-06-26 13:28:37,932 INFO    MainThread:45508 [wandb_init.py:init():606] backend started and connected
2023-06-26 13:28:37,935 INFO    MainThread:45508 [wandb_run.py:_label_probe_notebook():1235] probe notebook
2023-06-26 13:28:37,935 INFO    MainThread:45508 [wandb_run.py:_label_probe_notebook():1245] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-06-26 13:28:37,935 INFO    MainThread:45508 [wandb_init.py:init():703] updated telemetry
2023-06-26 13:28:37,971 INFO    MainThread:45508 [wandb_init.py:init():736] communicating run to backend with 60.0 second timeout
